scio-core/src/main/scala/com/spotify/scio/values/PairSkewedSCollectionFunctions.scala

```diff
@@ -60,13 +60,12 @@ object SkewedJoins {
 
   // some sensible defaults for skewed joins
   val DefaultHotKeyThreshold: Int = 9000
-  val DefaultHotKeyMethod: HotKeyMethod.Threshold = HotKeyMethod.Threshold(DefaultHotKeyThreshold)
```

Comment:
not used anywhere

Reply:
Was left here because people ca use those default when calling the new API (as overloading with default is not allowed). The default constants we not made private.

Reply:
Ok